src/main/kotlin/org/opensearch/commons/alerting/action/GetWorkflowAlertsResponse.kt

```diff
@@ -0,0 +1,51 @@
+package org.opensearch.commons.alerting.action
+
+import org.opensearch.common.io.stream.StreamInput
+import org.opensearch.common.io.stream.StreamOutput
+import org.opensearch.commons.alerting.model.Alert
+import org.opensearch.commons.notifications.action.BaseResponse
+import org.opensearch.core.xcontent.ToXContent
+import org.opensearch.core.xcontent.XContentBuilder
+import java.io.IOException
+import java.util.Collections
+
+class GetWorkflowAlertsResponse : BaseResponse {
+    val alerts: List<Alert>
+    val associatedAlerts: List<Alert>
+    // totalAlerts is not the same as the size of alerts because there can be 30 alerts from the request, but
+    // the request only asked for 5 alerts, so totalAlerts will be 30, but alerts will only contain 5 alerts
+    val totalAlerts: Int?
+
+    constructor(
+        alerts: List<Alert>,
+        associatedAlerts: List<Alert>,
+        totalAlerts: Int?
```

Comment:
why is `totalAlerts` nullable?

Reply:
this is a parallel api to get alerts. Keeping it consistent with the response structure of GetAlertsResponse which has totalAlerts as nullable

https://github.com/opensearch-project/common-utils/blob/main/src/main/kotlin/org/opensearch/commons/alerting/action/GetAlertsResponse.kt#L17